gensim/scripts/glove2word2vec.py

```diff
@@ -72,12 +72,20 @@ def glove2word2vec(glove_input_file, word2vec_output_file):
     # do the actual conversion
     num_lines, num_dims = glove2word2vec(args.input, args.output)
     logger.info('converted model with %i vectors and %i dimensions', num_lines, num_dims)
-
+    
     # test that the converted model loads successfully
     model = gensim.models.Word2Vec.load_word2vec_format(args.output, binary=False)
     logger.info('model %s successfully loaded', model)
     logger.info('testing the model....')
-    logger.info('top-10 most similar words to "king": %s', model.most_similar(positive=['king'], topn=10))
-    logger.info('similarity score between "woman" and "man": %s', model.similarity('woman', 'man'))
+   
+    try:
+        logger.info('top-10 most similar words to "king": %s', model.most_similar(positive=['king'], topn=10))
+        logger.info('similarity score between "woman" and "man": %s', model.similarity('woman', 'man'))
+    except:
+        logger.error('"king" may not be present in the vocab of the test file. checking for model file creation now....')
+        if os.path.isfile(os.path.join(os.getcwd(), args.output)):
```

Comment:
Why `getcwd`? This script has nothing to do with the "current working directory".
Use `args.output` only.